README.md

```diff
@@ -1,2 +1,55 @@
 # go-tilepacks
-A Go-based tile downloader that saves to deduplicated mbtiles files.
+
+A Go-based tile downloader that saves to deduplicated files.
```

Comment:
Is the `disk` outputter deduplicated?

Reply:
Not sure I understand. There is another patch to be written to check whether a tile needs to be cloned at all but since that's `tilepack` logic rather than `outputter` logic I just followed the same convention as the mbtiles outputter to "INSERT OR REPLACE" on disk.

Reply:
The mbtiles outputter has an indirection step, so it ends up with only one copy of the tile contents. First, it calculates the MD5 hash of the tile contents and then assumes that's a unique ID for the tile contents. Then it inserts (or replaces) into both the `images` and `map` tables. When we get tile contents that are duplicates, they should all get replaced over the same `images` row¹.

As far as I can tell, the `disk` outputter just seems to write each tile to a `"z/x/y.fmt"` file which isn't deduplicated. Was it the intention to add deduplication in a subsequent patch? (If not, I just thought maybe we should remove the word "deduplicated" from the synopsis line.)

1: It's possible it would be faster to check if the `images` row exists before inserting/replacing it, that's a TODO to look into.

Reply:
Should we just say:

> A Go-based tile downloader that saves to files, optionally deduplicated.

The deduplication is cool for MBTiles, but seems like a later enhancement for the `files` option.

Reply:
:+1: